lib/book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] The reason why the reviewer requested you to add the setter as well, is because here when you want to rent a book, the record will also be created for it. Currently, the association that is there would mean that we only want to "read" which makes it a wrong association between a book and a rental since we want to be able to both read and create rentals for a book.